_stories/molecules/Breadcrumbs/README.md

```diff
@@ -0,0 +1,134 @@
+# Breadcrumbs
+
+**NOTE**: The component will only render with a minimun width of **991px**
+
+The `<Breadcrumbs>` component creates navigation links for the current pathname based on a configuration object.
+
+[See the prototype here](https://ds.gumgum.com/stable/).
+
+## Props
+
+| name           | description                                                                                                                                                  | required |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| config         | Object that defines the app routes. See below for details.                                                                                                   | true     |
+| pathname       | Current pathname either from react-router or window.location.pathname                                                                                        | true     |
+| hideMenus      | Boolean attribute to prevent displaying subpath submenus                                                                                                     | false    |
+| linkElt        | Optional component to display as the breadcrumbs. Receives prop "to" as its href                                                                             | false    |
```

Comment:
`linkElt` is actually `LinkElt`. Though I suggest changing it. [See comment](https://github.com/gumgum/gumdrops/pull/9/files#r181261541)

Reply:
It is only LinkElt on the Breadcrumb component, not on the main component. But I will apply the suggestion.

Reply:
Updated to linkComponent